tests/unit/s2n_shutdown_test.c

```diff
@@ -35,8 +35,8 @@ int main(int argc, char **argv)
     BEGIN_TEST();
 
     const uint8_t close_notify_alert[] = {
-        2 /* AlertLevel = fatal */,
-        0 /* AlertDescription = close_notify */
+        S2N_TLS_ALERT_LEVEL_WARNING,
```

Comment:
Shouldn't this be alert level fatal?

Reply:
Nope: https://github.com/aws/s2n-tls/blame/e5d334e33f705487b501347085792c5222d095e6/tls/s2n_alerts.c#L253

I couldn't find a line in the RFC stating that the level should be warning, but you can see the message on the relevant commit: https://github.com/aws/s2n-tls/commit/b0b3871190e44f8e55e6ce6a16134af685cb01cc It also just makes sense that it would be a warning. Most implementations ([including ours](https://github.com/aws/s2n-tls/blame/e5d334e33f705487b501347085792c5222d095e6/tls/s2n_alerts.c#L222-L226)) probably ignore the level completely, but it'd be an easy implementation mistake to treat it as an arbitrary fatal error instead of a close_notify if the level was set to fatal.